client/src/pages/Plantkins.js

```diff
@@ -61,7 +88,10 @@ function Plants() {
         setUser(res.data[0]);
       })
       .catch((err) => console.log(err));
-  }, [user]);
+    
+      getUsers().then(({data})=> setUserList(data.filter(a=> !(a.user_id === userId))))
+    
+  }, []);
```

Comment:
just add [user] back into the dependency array

Reply:
How would I see that it is actually there.  Also, I would take out the console.logs on lines 73 and 56.

Reply:
There are now some conflicts.  I believe they were caused when master was last merged.